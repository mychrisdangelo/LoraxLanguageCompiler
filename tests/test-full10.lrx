/* 
 * Author: Kira Whitehouse
 * Stress testing empty brace edge case tree definition
 */

int main()
{
	print("hello world\n\n");

	print(1[2, 3, 4, 5[], 6, 7[]], "\n");

	print('a'['b'['c'[]]], "\n\n\n");

	print(true[true[false, true], false[]], "\n");

}