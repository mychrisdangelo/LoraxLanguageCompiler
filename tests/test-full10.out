hello world

1[2[null,null,null,null,null,null],3[null,null,null,null,null,null],4[null,null,null,null,null,null],5[null,null,null,null,null,null],6[null,null,null,null,null,null],7[null,null,null,null,null,null]]
abc


true[true[false[null,null],true[null,null]],false[null,null]]
